### PR TITLE
Add support for all SQLAlchemy pg drivers Fixes #8

### LINKED ISSUE #8: Support other drivers eg pg8000 or psycopg2cffi
Also do not depend on psycopg2

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

File: redshift_sqlalchemy/dialect.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 from sqlalchemy import schema, util, exc
 from sqlalchemy.dialects.postgresql.base import PGDDLCompiler
-from sqlalchemy.dialects.postgresql.psycopg2 import PGDialect_psycopg2
+from sqlalchemy.dialects.postgresql import (
```
Comment (graingert) [inline redshift_sqlalchemy/dialect.py]: this will probably break on sqlalchemy versions without these dialects.

**Comment by graingert** [comment]
I don't really like this solution as I think it might break people with older SQLAlchemy versions, also the tests don't seem great.

**Comment by a5m0** [comment]
Is there any new movement on getting psycopg2 hard requirement removed? Would like to use pg8000 because of licensing...

**Comment by graingert** [comment]
@a5m0 no movement happy to take a look at PRs though.

**Pull request closed without merging**